### ISSUE TITLE
Has anyone reproduced the Alexnet results reported on ILSVRC 2012 validation data?

### ISSUE THREAD

**Issue by jstudy** [vlfeat/matconvnet#635]
The pretrained model of matconvnet-alex is said to reach 41.8% top-1 err. and 19.2% top- 5 err on the ILSVRC 2012 validation data, with one single centre-crop evaluation. However, my validation error is much higher than the above results. I used the given alexnet of beta20, with 'simplenn' model, 'batchNormalization', 'gaussian' weightInitMethod , and 'cudnn5 ' on windows 8.  However, the validation errors are almost the same, about 44.8% top-1 err. and 21.8% top-5 err. Can anyone figure this out?
